lib/compute/jenkins-main-node.ts

```diff
@@ -375,7 +375,7 @@ export class JenkinsMainNode {
 
       // Commands are fired one after the other but it does not wait for the command to complete.
       // Therefore, sleep 60 seconds to wait for plugins to install and jenkins to start which is required for the next step
-      InitCommand.shellCommand('sleep 60'),
+      InitCommand.shellCommand('sleep 65'),
```

Comment:
Is this intentional? seems not documentation change.

Reply:
If necessary change, the comment above needs to be updated to reflect the new value as well.

Reply:
This is to trigger internal deployment. There was a hiccup in jenkins installation. Looks like a transient issue. This will retrigger the redeployment.